ONSdigital/response-operations-ui
# Remove bi/generate enrolment code using event

# Motivation and Context
New enrolment codes will now be generated using a case event rather than a the old POST endpoint. The UI has been updated to use this endpoint

# What has changed
Code now uses case event for generating new enrolment code
Can no longer generate a new enrolment code if there is an existing active one for that business/survey

# How to test?
`pipenv run python run_tests.py` should run tests and all pass
Run acceptance tests with associated PR's below
With associated PR's try and generate a new code through the UI and confirm that a new code is created and usable

# Links
[Trello](https://trello.com/c/npXH3s9x/16-retain-current-enrolment-code-functionality-enrolment-codes)
## PR's
[case-service](https://github.com/ONSdigital/rm-case-service/pull/51)
[casesvc-api](https://github.com/ONSdigital/rm-casesvc-api/pull/21)
[rasrm-acceptance-tests](https://github.com/ONSdigital/rasrm-acceptance-tests/pull/147)
